lib/sqlitex/sql_builder.ex

```diff
@@ -0,0 +1,74 @@
+defmodule Sqlitex.SqlBuilder do
```

Comment:
I don't know much about `Ecto`, but I know they have spent a lot of time on their API for creating SQL queries. I wonder how much value there is in making our own SQL generator vs doing the SQL creation at the Ecto layer?